src/main/java/org/snomed/snowstorm/core/data/services/BranchMetadataHelper.java

```diff
@@ -69,6 +69,14 @@ public static Map<String, Set<String>> getRebaseDuplicatesRemoved(Commit commit)
 		return (Map<String, Set<String>>) commit.getBranch().getMetadata().getAsMap().computeIfAbsent(key, k -> new HashMap<String, Set<String>>());
 	}
 
+	public static void setRebaseDuplicatesRemoved(Commit commit, Class<?> clazz, Set<String> duplicateComponentIds) {
+		if (commit == null || clazz == null || duplicateComponentIds == null || duplicateComponentIds.isEmpty()) {
```

Comment:
@dmcgihtsdo This is a bad idea. You could just add items to the map after getRebaseDuplicatesRemoved

Reply:
@dmcgihtsdo It would be cleaner if you just add these items directly into the map. It is a bad idea to add setters in a helper class.

Reply:
Great, I have done this.